core/server/worker/src/main/java/alluxio/worker/netty/ShortCircuitBlockReadHandler.java

```diff
@@ -157,15 +155,7 @@ public void exceptionCaught(Throwable e) {
             ctx.writeAndFlush(
                 RPCProtoMessage.createResponse(AlluxioStatusException.fromThrowable(e)));
           }
-
-          @Override
-          public String toString() {
-            return String.format("OpenBlock: Session=%d, Request:=%s", mSessionId, request
-                .toString());
-          }
-        });
-      }
-    });
+        }, "OpenBlock", "OpenBlock: Session=%d, Request=%s", mSessionId, request.toString()));
```

Comment:
Updated, thanks. Inlining the method name makes more sense to me, since there is no restriction that the description must start with the method name. I only use the method name variable when the method name is ambiguous.